projects/common/lib/components/checkbox/checkbox.component.html

```diff
@@ -0,0 +1,11 @@
+
+<input #checkbox type="checkbox" class="input-lg form-check-input" name="{{objectId}}" 
```

Comment:
It's a small point, but can you please try and prefix all objectId's in the templates? For example `name_{{objectId}})`
```suggestion
<input #checkbox type="checkbox" class="input-lg form-check-input" name="name_{{objectId}}" 
```

Reply:
If you miss a few of these it's fine and I can add them after, it's not a huge priority, but it's something the test team often asks for.